webofneeds/won-core/src/main/java/won/protocol/repository/MessageEventRepository.java

```diff
@@ -19,6 +19,13 @@
 
   List<URI> getMessageURIsByParentURI(URI parentURI);
 
+  List<MessageEventPlaceholder> findByParentURIAndMessageType(URI parentURI, WonMessageType messageType);
+
+  @Query("select msg from MessageEventPlaceholder msg where msg.parentURI = :parent and " +
+    "referencedByOtherMessage = false")
+  List<MessageEventPlaceholder> findByParentURIAndReferencedByOtherMessage(
```

Comment:
is the naming right here? shouldnt this be "findByParentURIAndNotReferencedByOtherMessage" => not referenced cause referencedByOtherMessage = false?